VocaDb.ResXFileCodeGenerator/FileOptions.cs

```diff
@@ -0,0 +1,114 @@
+﻿using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.Diagnostics;
+
+namespace VocaDb.ResXFileCodeGenerator;
+
+public sealed record FileOptions
+{
+	public string InnerClassInstanceName { get; init; }
+	public string InnerClassName { get; init; }
+	public InnerClassVisibility InnerClassVisibility { get; init; }
+	public bool PartialClass { get; init; }
+	public bool StaticMembers { get; init; } = true;
```

Comment:
Can you please add comments for these options as the number of options has been increasing? Copying and pasting the description from readme should be enough.